src/k8s/pkg/k8sd/features/localpv/chart.go

```diff
@@ -20,11 +20,11 @@ var (
 	ImageTag = "0.8.2-ck3"
 
 	// csiNodeDriverImage is the image to use for the CSI node driver.
-	csiNodeDriverImage = "ghcr.io/canonical/csi-node-driver-registrar:2.11.1-ck7"
+	csiNodeDriverImage = "ghcr.io/canonical/csi-node-driver-registrar:3347350ff6d84b5544f75b67d25200997c5fd6c67affe48feb374fbda1dffae1-amd64"
 	// csiProvisionerImage is the image to use for the CSI provisioner.
-	csiProvisionerImage = "ghcr.io/canonical/csi-provisioner:5.0.2-ck1"
+	csiProvisionerImage = "ghcr.io/canonical/csi-provisioner:cf6224e54904e45fb18cbcde368789e86ea9cddaada559b843da7b8c63397d99-amd64"
 	// csiResizerImage is the image to use for the CSI resizer.
-	csiResizerImage = "ghcr.io/canonical/csi-resizer:1.11.2-ck1"
+	csiResizerImage = "ghcr.io/canonical/csi-resizer:2b3ce9bde190654dae7639f57a1528c79e09913b8bdf6a5dc9e711811c7c076d-amd64"
```

Comment:
Required changes in:
https://github.com/kubernetes-csi/external-resizer/compare/v1.11.2...canonical:external-resizer:v1.11.2-ck0